### PR TITLE
Building submission model

### PR DESCRIPTION
**Pull request by ashleiperry**

### CODE REVIEW

File: spec/models/submission_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Submission, type: :model do
+it {should have_valid(:title).when("Warning this thing is hot")}
```
Comment (smkopp92) [inline spec/models/submission_spec.rb]: I would indent lines 4-15

File: spec/models/submission_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+it {should have_valid(:user_id).when(1)}
+it {should_not have_valid(:user_id).when(nil, "")}
+
+it {should have_valid(:description).when("Crazy lady sets sofa on fire")}
+it {should have_valid(:description).when(nil, "")}
+
+
+it {should have_valid(:url).when(nil, "")}
+it {should have_valid(:url).when("www.example.com")}
+
```
Comment (smkopp92) [inline spec/models/submission_spec.rb]: I would remove all this white space below line 15

File: app/models/submission.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Submission < ApplicationRecord
+  validates :title, presence: true
+  validates :user_id, presence: true
+
```
Comment (smkopp92) [inline app/models/submission.rb]: I would remove this whitespace :)

File: app/views/submissions/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<ul>
+<% @submissions.each do |submission| %>
```
Comment (smkopp92) [inline app/views/submissions/index.html.erb]: Lines 2-4 should be indented further

File: spec/controllers/submissions_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (smkopp92) [inline spec/controllers/submissions_controller_spec.rb]: I would remove any tests that don't actually test anything!

File: spec/factories/keywords.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (smkopp92) [inline spec/factories/keywords.rb]: Setting up factory girl is alright, but please try to not have empty factories set up!

File: spec/helpers/submissions_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (smkopp92) [inline spec/helpers/submissions_helper_spec.rb]: Please refrain from having empty test files. Just delete the file entirely if you're never going to have a submission helper.

File: spec/features/sees_submissions_spec.rb [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+feature "sees submissions" do
+  scenario "see all the submissions" do
+    submission = Submission.create(title: "Not stunting in my Air Force Ones", user_id: 1)
+    another_submission = Submission.create(title: "This product caused my divorce", user_id: 1)
+
+    visit submissions_path
+    expect(page).to have_content(submission.title)
+    expect(page).to have_content(another_submission.title)
+
```
Comment (jdmacmurtrie) [inline spec/features/sees_submissions_spec.rb]: Please delete whitespace.

**Pull request merged**